### PR TITLE
Stats notifications [ MAILPOET-1571]

### PR DESCRIPTION
**Pull request by pavel-mailpoet**

### CODE REVIEW

File: views/emails/statsNotification.html [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+                            </tr></table>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td class="mailpoet_padded_bottom mailpoet_padded_side" valign="top" style="border-collapse:collapse;padding-bottom:20px;padding-left:20px;padding-right:20px">
+                          <div>
+                            <table width="100%" cellpadding="0" cellspacing="0" border="0" style="border-spacing:0;mso-table-lspace:0;mso-table-rspace:0;border-collapse:collapse">
+                              <tr>
+                                <td class="mailpoet_button-container" style="text-align:center;border-collapse:collapse">
+                                  <a class="mailpoet_button" href="https://account.mailpoet.com/premium" style="display:inline-block;-webkit-text-size-adjust:none;mso-hide:all;text-decoration:none;text-align:center;background-color:#fe5301 ;border-color:#0074a2 ;border-width:0px ;border-radius:3px ;border-style:solid ;width:288px ;line-height:50px ;color:#ffffff ;font-family:Arial, 'Helvetica Neue', Helvetica, sans-serif ;font-size:20px ;font-weight:normal ">
```
Comment (wxa) [inline views/emails/statsNotification.html]: According to the specs the link should point to `{plugin_url}?page=mailpoet-premium`.

File: lib/Cron/Workers/StatsNotifications/Worker.php [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    $unsubscribed = ($newsletter->statistics['unsubscribed'] * 100) / $newsletter->total_sent;
+    $context =  [
+      'subject' => $newsletter->subject,
+      'preheader' => sprintf(_x(
+        '%1$s%% opens, %2$s%% clicks, %3$s%% unsubscribes in a nutshell.', 'newsletter open rate, click rate and unsubscribe rate', 'mailpoet'),
+        number_format($clicked, 2),
+        number_format($opened, 2),
+        number_format($unsubscribed, 2)
+      ),
+      $context['topLinkClicks'] = 0,
```
Comment (wxa) [inline lib/Cron/Workers/StatsNotifications/Worker.php]: It's the first time I see such a recursive array definition, had to trace it in mind to check it's okay. How about making it more readable?

Comment (pavel-mailpoet) [reply lib/Cron/Workers/StatsNotifications/Worker.php]: Sorry, this is my typo, I didn't mean to write it this way. It is a copy & paste error.

File: views/emails/statsNotification.html [hunk 3]
```diff
@@ -0,0 +1,452 @@
...
+                            <%= __('Most clicked link') %>
+                          </span>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td class="mailpoet_text mailpoet_padded_bottom mailpoet_padded_side" valign="top" style="word-break:break-word;word-wrap:break-word;padding-top:0;border-collapse:collapse;padding-bottom:20px;padding-left:20px;padding-right:20px">
+                          <table style="border-spacing:0;mso-table-lspace:0;mso-table-rspace:0;border-collapse:collapse" width="100%" cellpadding="0">
+                            <tr>
+                              <td class="mailpoet_paragraph" style="word-break:break-word;word-wrap:break-word;text-align:center;border-collapse:collapse;color:#000000;font-family:Arial,'Helvetica Neue',Helvetica,sans-serif;font-size:16px;line-height:25.6px">
+                                <a href="<%= topLink %>" target="_blank" rel="noopener noreferrer" style="color:#008282;text-decoration:underline">
```
Comment (wxa) [inline views/emails/statsNotification.html]: If the most clicked link is a shortcode, we display it as-is and the link goes nowhere. On the stats page we replace shortcodes with a text line such as "Manage subscription link" and they're not clickable. Should we do the same or it's too much hassle for nothing?

**Comment by pavel-mailpoet** [comment]
It works locally. So there must be something different on your testing website that breaks it. Could you enable WP_DEBUG and try again? If there is an exception thrown it should log it to a debug log. Also, for this test, you don't need to wait 24 hours. You can update the `scheduled_at` in the database to be in the past and it will pick it up.

**Pull request merged**